### ISSUE TITLE
Tailwind UI's color config is missing the "current" versions from standard Tailwind

### ISSUE THREAD

**Issue by brandonpittman** [tailwindlabs/tailwindui-issues#232]
⤴️

bg-current...
text-current..
border-current...

All missing because Tailwind UI overwrites the default Tailwind config.

**Comment by adamlogic** [comment]
I have no idea if this is wise, but here's how I'm working around this issue:

```js
const defaultTheme = require("tailwindcss/defaultTheme");

module.exports = {
  theme: {
    colors: {
      ...defaultTheme.colors,
      ...require("@tailwindcss/ui")().config.theme.colors,
    },
```

This ensures that the Tailwind UI color palette has priority, but I don't *lose* any colors (such as "current") from the default theme.

**Comment by brandonpittman** [comment]
This was fixed by v0.6. Closing.

**Comment by adamlogic** [comment]
I was still having this issue in v0.6.0 FWIW. I have a workaround, though, so I'm fine to leave it closed.

Do you know if there are release notes for the @tailwindcss/ui plugin?
